README.md

```diff
@@ -130,10 +119,9 @@ docker run -d --name addis \
   -e ADDIS_CORE_OAUTH_GOOGLE_SECRET=<google-secret> \
   -e ADDIS_CORE_OAUTH_GOOGLE_KEY=<google-key> \
   -e TRIPLESTORE_BASE_URI=https://jena-es:3030 \
-  -e KEYSTORE_PATH=/ssl/keystore.jks \
-  -e KEYSTORE_PASSWORD=develop \
   -e EVENT_SOURCE_URI_PREFIX=https://jena-es:3030 \
   -e CLINICALTRIALS_IMPORTER_URL=https://nct.drugis.org \
+  -e PATAVI_API_KEY=someapikeyhere \
```

Comment:
Earlier in the readme sometokenhere is used, I think we should keep in consistent.